index.php

```diff
@@ -43,7 +43,6 @@
 }
 
 const PATH = '/';
-const ROOT_PATH = __DIR__;
```

Comment:
Should stay as deprecated, this being a huge breaking change for all templates and modules